src/events/updateGuilds.js

```diff
@@ -0,0 +1,40 @@
+let timeout;
+
+module.exports = {
+  name: 'updateGuilds',
+  execute: async (forceUpdate = false, client) => {
```

Comment:
acho que nao da pra fazer isso pq quando voce chamar `client.emit('updateGuilds')` o event handler vai detectar e chamar `event.execute(client)` (ja que nao tem argumentos pra colocar antes) e o argumento `client` vai ser entendido como o parametro `forceUpdate`
```suggestion
  execute: async (forceUpdate, client) => {
```

Reply:
Eu testei e isso não acontece:
```js
const { EventEmitter } = require("node:events");

const events = new EventEmitter();

events.on("test", (...args) => handler(args, { user: "client" }));

const handler = (args, client) => {
	console.log("file: test.js ~ line 10 ~ handler ~ args", args);
	console.log("file: test.js ~ line 10 ~ handler ~ client", client);
};

events.emit("test");
events.emit("test", "com argumentos");
```
![image](https://user-images.githubusercontent.com/32278696/201254261-230b2236-d187-4e13-8ad1-fe3389bf4128.png)